framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -83,10 +84,18 @@ SideSetsAroundSubdomain::modify()
   // Get a reference to our BoundaryInfo object for later use
   BoundaryInfo & boundary_info = mesh.get_boundary_info();
 
+  // Prepare to query about sides adjacent to remote elements if we're
+  // on a distributed mesh
+  const processor_id_type my_n_proc  = mesh.n_processors();
+  const processor_id_type my_proc_id = mesh.processor_id();
```

Comment:
For future reference, we are fully in support of `auto` for these kinds of simple function returns.